### DISCUSSION TITLE
401 Bad credentials error upon install

### DISCUSSION THREAD

**Discussion by mubassirgamfoi** [projectdiscovery/pdtm#108] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/42926244/222971862-2daa9969-4340-4862-b8ac-777e47a7f14e.png)

**Comment by xm1k3** [comment]
Hii @Mubspatel, I manually tried to see the individual tool requests and it turns out that they go with status code 200, can I ask you to try again?

If you have the same 401 response can I ask you for more info, for example the command you launched?

![image](https://user-images.githubusercontent.com/73166077/224693646-a9fec0c6-bd4f-4a25-9f17-4b9b990308be.png)

**Comment by ehsandeep** [comment]
@Mubspatel Can you retry with latest released version?

**Comment by tarunKoyalwar** [comment, marked as answer]
@xm1k3  could be this https://discord.com/channels/695645237418131507/1087452781918552094/1088163904347197440
i.e
>@Elkomy this happens because go-github whenever available tries to use github token from GITHUB_TOKEN env variable (to avoid ratelimits and stuff) . It looks like github token you use currently is expired/revoked.
unset/remove $GITHUB_TOKEN or replace it with new token

**Comment by mubassirgamfoi** [comment]
You're Tarun, The issue was occurring because of $GITHUB_TOKEN. Thanks for
the help, Tarun and Sandeep.
